docs/getting-started/power-setup.md

```diff
@@ -28,3 +28,27 @@ defaultRootCommand: "exec"
 
 With this setup, calling `sn` will yield the same result as `snipkit exec`. If you want to call
 the `print` command instead, type `sn print`.
+
+### Interactive ZSH Widget
```

Comment:
This is very specific to your setup and needs. I would rather have just the `--output` flag documented in `Overview > Print snippets` as a tip.